src/features/focused_geometry_layer/renderers/FocusedGeometryRenderer.ts

```diff
@@ -67,140 +61,8 @@ const getLayersConfig = (
   ];
 };
 
-export class FocusedGeometryRenderer extends LogicalLayerDefaultRenderer {
-  public readonly id: string;
-  private sourceId: string;
-  private layerConfigs: maplibregl.LayerSpecification[] = [];
-  private availableIcons: Set<string> = new Set();
-
+export class FocusedGeometryRenderer extends UserGeometryRenderer {
```

Comment:
Note: I moved out part of [FocusedGeometryRenderer](https://github.com/konturio/disaster-ninja-fe/pull/721/files#diff-2411c4a21c676b90ee0522b8065cdf49365c3b9bcb03d2d13036953f3178a20c) code into common [UserGeometryRenderer](https://github.com/konturio/disaster-ninja-fe/pull/721/files#diff-e12b4fa823312fd4ad544337b0ffb661d2a6256aefa64ff56c61e4c9e519bde0) to avoid code duplication inside ReferenceAreaRenderer